src/main/compassus/core.cljc

```diff
@@ -17,7 +17,7 @@
   (-> app :config :root-class))
 
 (defn- make-root-class
-  [{:keys [routes wrapper history]}]
+  [{:keys [routes wrapper wrapper-class history]}]
```

Comment:
we can probably get at the query by invoking `om.next/react-type` on the wrapper thing and checking if it implements a query with `om.next/iquery?`